### DISCUSSION TITLE
Problem with verification

### DISCUSSION THREAD

**Discussion by mannuccilucia** [RosettaCommons/rosetta#279] [ROSIE]
Hello,
I am a researcher at the Università Politecnica delle Marche (UnivPM, Ancona, Italy). I kindly request that you add our institute’s email domain to your list of verified academic email addresses.
The domain is staff.univpm.it

Thanks in advance

**Comment by RezaHasani7676** [comment]
Tabriz University of Medical Sciences
 tbzmed.ac.ir

**Reply by roccomoretti** [reply]
Unfortunately, there are legal issues with using servers in the United States to provide computational services to people in Iran, which means that we are unable to approve Iranian email addresses for ROSIE accounts at this time.

We apologize for the current state of the world.

**Comment by lyskov** [comment]
> Hello, I am a researcher at the Università Politecnica delle Marche (UnivPM, Ancona, Italy). I kindly request that you add our institute’s email domain to your list of verified academic email addresses. The domain is staff.univpm.it
> 
> Thanks in advance

— Thank you for letting us know! This should be fixed now, if you still have issues logging-in please feel free to re-open this issue.
